.github/workflows/equivalence-test-update.yml

```diff
@@ -18,14 +18,14 @@ jobs:
       - name: target_branch
         id: target_branch
         run: |
-          merged = ${{ github.event.pull_request.merged }}
-          target_branch = ${{ github.event.pull_request.base.ref }}
+          merged=${{ github.event.pull_request.merged }}
+          target_branch=${{ github.event.pull_request.base.ref }}
```

Comment:
We might want to enclose this in (single) quotes in case the branch name has spaces or something that needs escaping.

Reply:
Done!